lib/logger.js

```diff
@@ -0,0 +1,66 @@
+const logger = (() => {
+  const checkIfLogsEnabled = () => {
+    const search = window?.location?.search
+    const enabled = search && search.includes('debug=true')
+    global.areLogsEnabled = enabled || false
+    return global.areLogsEnabled
+  }
+
+  const isDev = process.env.NODE_ENV !== 'production'
+
+  const print = (type, ...messages) => {
+    if (typeof global.areLogsEnabled === 'undefined') {
+      checkIfLogsEnabled()
+    }
+
+    if (global.areLogsEnabled || isDev) {
+      switch (type) {
+        case 'info':
+          console.info(
+            '%c Info:',
+            'background: blue; color: white;',
```

Comment:
Changing the color every time can be more overwhelming than helpful; I'd suggest errors get highlighted, warnings get text color change, and the rest are default (no) styling.

Reply:
Done! :)